webstruct/text_tokenizers.py

```diff
@@ -109,7 +109,17 @@ def tokenize(self, text):
 class DefaultTokenizer(WordTokenizer):
     def tokenize(self, text):
         tokens = super(DefaultTokenizer, self).tokenize(text)
-        # remove standalone commas and semicolons
+        """
```

Comment:
Could you please use a block comment, as per pep8 (i.e. start each line with `#`)? Or make this an actual docstring.